src/test/scala/com/microsoft/hyperspace/index/rules/FilterIndexRuleTest.scala

```diff
@@ -157,7 +158,7 @@ class FilterIndexRuleTest extends HyperspaceRuleTestSuite {
       bucketSpec: Option[BucketSpec]): Unit = {
     val allIndexes = IndexCollectionManager(spark).getIndexes(Seq(Constants.States.ACTIVE))
     val expectedLocation = getIndexDataFilesPath(indexName)
-    assert(location.rootPaths.head.equals(expectedLocation))
+    assert(location.rootPaths.head.toString.startsWith(expectedLocation.toString))
```

Comment:
ditto.

looks like `rootPath` will return the actual "file" paths if `FileIndex` is created with actual files.

Reply:
thanks @imback82 , updated this test so now it checks all files